cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -1443,6 +1446,11 @@ private void setDataVolumesClaimsAndMountPaths(Storage storage) {
                 KafkaListenerAuthenticationOAuth oauth = (KafkaListenerAuthenticationOAuth) listener.getAuth();
                 volumeList.addAll(AuthenticationUtils.configureOauthCertificateVolumes("oauth-" + ListenersUtils.identifier(listener), oauth.getTlsTrustedCertificates(), isOpenShift));
             }
+
+            if (isListenerWithCustomAuth(listener)) {
+                KafkaListenerAuthenticationCustom custom = (KafkaListenerAuthenticationCustom) listener.getAuth();
+                volumeList.addAll(AuthenticationUtils.configureGenericSecretVolumes("custom-" + ListenersUtils.identifier(listener), custom.getSecrets(), isOpenShift));
```

Comment:
Can we fit something like `custom-listener-` here? Eventually some shortcut of it? `custom` sounds a bit too generic => it might prove to be limiting int he future, but it will not be easy to change if already used by users.